common/scylla_image_setup

```diff
@@ -7,13 +7,10 @@
 import os
 import sys
 import pathlib
-from lib.scylla_cloud import get_cloud_instance, is_gce, is_redhat_variant
+from lib.scylla_cloud import get_cloud_instance, is_gce
```

Comment:
I think we should still have the ability to install machine image on rpm based OS and run `scylla_io_setup` for example. we recently had a customer that worked on `Rhel8` and installed it manually 

Let's drop only the code related directly to building image based on rpm